src/util/__tests__/createServerRootMixin.test.js

```diff
@@ -52,7 +52,6 @@ describe('createServerRootMixin', () => {
         createSSRApp({
           mixins: [
             createServerRootMixin({
-              renderToString,
```

Comment:
we could still make the signature to `importRenderToString: () => import('render-to-string')` that should be tree-shaken in mai

Reply:
It's imported only on server and from `entry-server.js`. So I'm not sure how that'd help with tree-shaking. And even if we make the signature like that, it'd be better to pass it directly to `findResultsState`, because previously I had to pass `renderToString` all the way from createServerRootMixin to augmentInstantSearch → ...